README.md

```diff
@@ -24,6 +28,32 @@ If you have Go developer tools installed, you can install it with `go install`:
 $ go install github.com/stateful/runme@latest
 ```
 
+## Commands
```

Comment:
```suggestion
## Commands

```

Let's keep the same style: new line after headings and always before and after a code block.